src/components/PageDiscord.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
```

Comment:
Should be able to get rid of this completely (see comment re routing in RehydrationProvider)